### PR TITLE
doc/intro: add void linux to the list

### PR DESCRIPTION
**Pull request by lasers**
This adds `Void Linux` to the list.

### CODE REVIEW

**Comment by tobes** [comment]
I think this PR shows that tables in reStructuredText are a broken idea.  I think we can display this info in a way that is more readable and maintainable long term.  Reformatting things to add a line is like crazy

File: doc/intro.rst [hunk 1]
```diff
@@ -41,23 +41,23 @@ Philosophy
...
-|**Debian & Ubuntu**|``$ pypi-install py3status`` |Usually in `python-stdeb` package.       |
-+-------------------+-----------------------------+-----------------------------------------+
-|**Fedora**         |``$ dnf install py3status``  |                                         |
-+-------------------+-----------------------------+-----------------------------------------+
-|**Gentoo Linux**   |``$ emerge -a py3status``    |                                         |
-+-------------------+-----------------------------+-----------------------------------------+
-|**Pypi**           |``$ pip install py3status``  |                                         |
-+-------------------+-----------------------------+-----------------------------------------+
-
-Installation Note: This list is incomplete. Please help us fill in the gaps.
```
Comment (ultrabug) [inline doc/intro.rst]: I think we should encourage people to add more and contribute to the docs, please leave this or rephrase it.

Comment (lasers) [reply doc/intro.rst]: I put it there. I think this makes us look a bit desperate... which we aren't.... especially with the `Please help us fill in the gaps` message. If the users want to add something to this list, they can make a pull request... or if they don't... we can look into it ourselves.

Also, we should look into making a `man` page for `py3status`.... and next, make `py3status` available in many distributions (Debian, Ubuntu, ArchLinux, Gentoo, etc)... as I find that they're officially not in there.

Comment (ultrabug) [reply doc/intro.rst]: lol I don't think this looks desperate but alright it's no big deal anyway

**Pull request merged**